lib/labels_web/controllers/page_controller.ex

```diff
@@ -33,14 +35,28 @@ defmodule LabelsWeb.PageController do
              owner: target_owner,
              repo_name: target_repo
            }) do
+      # create ok changeset -> keep source info, remove target info
       conn
       |> put_flash(:info, "Labels synched!")
-      |> redirect(to: Routes.page_path(conn, :index))
+      |> render(
+        "index.html",
+        [repositories: repositories] ++
+          form_values(source_owner: source_owner, source_repo: source_repo)
```

Comment:
When sync is ok, we only want to keep the source owner/repo in the form and clear the target infos.